pyproject.toml

```diff
@@ -1,12 +1,16 @@
 [build-system]
-requires = ["setuptools>=61.0"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.wheel]
+packages = ["charms"]
 
 [project]
-name = "charm-lib-kubernetes-snaps"
+name = "charms.kubernetes-snaps"
 version = "0.0.0"
```

Comment:
can we have dynamic versioning? or do we like this?

Reply:
https://github.com/charmed-kubernetes/charm-lib-contextual-status/blob/main/pyproject.toml#L11

Reply:
Sure we can. Done!